go.sum

```diff
@@ -0,0 +1,230 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I think this sum file is too messy, and it don't just include your needed dependencies. you can delete it and rebuild the project to generate it again.

Reply:
this is done in #2, removed go.sum